erpnext/assets/doctype/asset_category/asset_category.py

```diff
@@ -29,7 +29,8 @@ def get_asset_category_account(fieldname, item=None, asset=None, account=None, a
 				account=None
 
 		if not account:
-			asset_category, company = frappe.db.get_value("Asset", asset, ["asset_category", "company"])
+			#if it is not finding the value it is returning single 'None' which is not iterable to asset_category and company.
+			asset_category, company = frappe.db.get_value("Asset", asset, ["asset_category", "company"]) if frappe.db.get_value("Asset", asset, ["asset_category", "company"]) else None, None
```

Comment:
This is very bad way to writing code, you are doing get_value twice. Instead you should set the result in a variable.

Reply:
done